### DISCUSSION TITLE
No Screenshots for achieving lvl 99

### DISCUSSION THREAD

**Discussion by iLight-24** [runelite/runelite#17151] [Q&A] [answered]
Hi,
I achieved 2 99's last week but when I tried to see the screenshots in my folder it was missing.
All other lvls screenshot was there and the 99 missing.

**Comment by Nightfirecat** [comment]
If you were in combat or some other situation which would have prevented the dialog from appearing, the screenshot plugin will not have seen anything to take a screenshot of.

**Reply by iLight-24** [reply]
I think the dialog for 99 is changed, As you can see in the code the pattern is not matched.

	private static final Pattern LEVEL_UP_PATTERN = Pattern.compile(".*Your ([a-zA-Z]+) (?:level is|are)? now (\\d+)\\.");
	private static final Pattern LEVEL_UP_MESSAGE_PATTERN = Pattern.compile("Congratulations, you've just advanced your ([a-zA-Z]+) level. You are now level (\\d+)\\.");
	
![image](https://github.com/runelite/runelite/assets/61458342/69301f84-af5d-4857-b9c7-810ae022318b)

**Reply by Nightfirecat** [reply]
Do you have level-up dialogs disabled?

**Reply by iLight-24** [reply]
Yes.

**Reply by Nightfirecat** [reply, marked as answer]
This is a bug, I've created #17152 to track it.
